terraform-infra-approvals/sscs-shared-infrastructure.json

```diff
@@ -0,0 +1,6 @@
+{
+  "resources": [
+    {"type": "azurerm_template_deployment"}
```

Comment:
This should really be wrapped in a module for log-alerts like the metric alert...
https://github.com/hmcts/sscs-shared-infrastructure/blob/4f87bcf2dd364152544a1d33ca66adb9fda674d6/alerts.tf#L30

Reply:
No idea what this means. @chris-moreton do you have any ideas about this?